### DISCUSSION TITLE
Viscosity distribution and wall shear stress around the cylinder

### DISCUSSION THREAD

**Discussion by pratyush507** [fppimenta/rheoTool#101] [General]
I am facing issue in getting viscosity distribution and wall shear stress calculation using the rheotool solver. How can i generate that ca you please help ?

**Comment by pratyush507** [comment]
Please help me to get the wall shear stress distribution and viscosity distribution while using rheotool as the wall shear stress utility is compactible with turbulence model not with rheotool . Please help

**Comment by fppimenta** [comment]
https://github.com/fppimenta/rheoTool/discussions/98#discussioncomment-4490548
